lib/active_admin/mongoid/filter_form_builder.rb

```diff
@@ -2,7 +2,12 @@ class ActiveAdmin::Filters::FormBuilder
 
   def filter(method, options = {})
     if method.present? && options[:as] ||= default_input_type(method)
-      template.concat input(method, options)
```

Comment:
Each branch of if performs the same operation. Could not that line be left as it was?